pkg/reports/reports.go

```diff
@@ -0,0 +1,86 @@
+package reports
+
+import (
+	"github.com/jetstack/preflight/api"
+	"github.com/jetstack/preflight/pkg/packaging"
+	"github.com/jetstack/preflight/pkg/results"
+	"github.com/jetstack/preflight/pkg/rules"
+	"github.com/jetstack/preflight/pkg/version"
+)
+
+// ConstructReport creates a report from a policy manifest and a results collection
+func ConstructReport(pm *packaging.PolicyManifest, rc *results.ResultCollection) (api.Report, error) {
```

Comment:
Is there any reason this is called `ConstructReport` rather than `Create Report`?

Reply:
I guess I'm seeing this as a replacement for an oop constructor method.

create works too.

I'm wondering if `NewReport` might be more idiomatic :man_shrugging:

Reply:
I've renamed it to new in the upcoming changes I was preparing to add on top here https://github.com/charlieegan3/preflight/pull/2

Reply:
Brought that in for this branch here: b47c45d70086844584894c93a1f17fc0d318a8a1